src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/AssessmentController.kt

```diff
@@ -66,35 +64,25 @@ class AssessmentController(
     val user = userService.getUserForRequest()
     val resolvedSortDirection = sortDirection ?: SortDirection.asc
     val resolvedSortBy = sortBy ?: AssessmentSortField.assessmentArrivalDate
+    val domainSummaryStatuses = statuses?.map { assessmentTransformer.transformApiStatusToDomainSummaryState(it) } ?: emptyList()
 
     val (summaries, metadata) = when (xServiceName) {
       ServiceName.cas2 -> throw UnsupportedOperationException("CAS2 not supported")
       ServiceName.temporaryAccommodation -> {
-        val summaries = if (crn != null) {
-          assessmentService.getAssessmentSummariesByCrnForUserCAS3(user, crn, xServiceName)
-        } else {
-          assessmentService.getVisibleAssessmentSummariesForUserCAS3(user)
+        val (summaries, metadata) = assessmentService.getAssessmentSummariesForUserCAS3(
+          user,
+          crn,
+          xServiceName,
+          domainSummaryStatuses,
+          PageCriteria(resolvedSortBy, resolvedSortDirection, page, perPage ?: defaultPageSize),
+        )
+        val transformSummaries = when (sortBy) {
+          AssessmentSortField.personName -> transformDomainToApi(user, summaries).sort(resolvedSortDirection, sortBy)
+          else -> transformDomainToApi(user, summaries)
         }
-
-        var filteredSummaries = transformDomainToApi(user, summaries)
-          .sort(resolvedSortDirection, resolvedSortBy)
-          .filterByStatuses(statuses)
```

Comment:
can we remove these sort and filer methods now?

Reply:
We(Cas3) dont store the person name in the table, so currently personName sort is using the existing sort functionality.

Reply:
Ah yes, sorry i didn't spot you were still calling the sort method

Reply:
removed un-used methods